SUPPORT.md

```diff
@@ -0,0 +1,36 @@
+# Getting help with git-rstudio-basics
+
+Thanks for using git-rstudio-basics!
+Before filing an issue, there are a few places to explore and pieces to put together to make the process as smooth as possible.
+
+## Make a reprex
+
+Start by making a minimal **repr**oducible **ex**ample using the  [reprex](https://reprex.tidyverse.org/) package. 
+If you haven't heard of or used reprex before, you're in for a treat! 
+Seriously, reprex will make all of your R-question-asking endeavors easier (which is a pretty insane ROI for the five to ten minutes it'll take you to learn what it's all about). 
+For additional reprex pointers, check out the [Get help!](https://www.tidyverse.org/help/) section of the tidyverse site.
+You can also check out the [How to make a reproducible R code example](https://community.appliedepi.org/t/how-to-make-a-reproducible-r-code-example/167/1) of the AppliedEpi community site.
+
+## Where to ask?
+
+Armed with your reprex, the next step is to figure out [where to ask](https://www.tidyverse.org/help/#where-to-ask). 
```

Comment:
This link should probably be updated or removed.

Reply:
Yep, I move it from the top to the last line about asking to the community https://github.com/epiverse-trace/git-rstudio-basics/pull/3/commits/bef3b35ee3297fd89eaf1b1a54560bb84d44eca4